packages/react-components/src/localization/locales/fi-FI/strings.json

```diff
@@ -379,4 +379,4 @@
     "leftNavButtonAriaLabel": "edellinen sivu",
     "rightNavButtonAriaLabel": "seuraava sivu"
   }
-}
\ No newline at end of file
+}

```

Comment:
Not sure if we need to merge all strings changes...

Reply:
Oh strange, maybe the linter added these new lines. I can remove them

Reply:
it's probably alright then